packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/tile_overlay_updates.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart' show setEquals;
+
+import 'utils/tile_overlay.dart';
+import 'types.dart';
+
+/// Represents the details of an update events of [TileOverlay]s.
```

Comment:
Maybe: Update specification for a set of [TileOverlays].

Reply:
done